test/TestCase.php

```diff
@@ -75,7 +77,7 @@ public function getSqlMock($key)
      * Returns a test double for the specified class.
      *
      * @param string $originalClassName
-     * @return \PHPUnit_Framework_MockObject_MockObject
+     * @return MockObject
      * @throws \PHPUnit_Framework_Exception
```

Comment:
Shall we do the same for \PHPUnit_Framework_Exception?

Reply:
It is the only one occurrence of this class i found.

Reply:
Ok, so forget about it...